bin/lib/get-defaults.js

```diff
@@ -43,13 +45,20 @@ function getDefaults (workPath, isEnterprise, callback) {
     if (!log) {
       return callback(new Error('CHANGELOG.md does not contain any versions'))
     }
-    if (log.version !== pkg.version) {
-      var errStr = 'CHANGELOG.md out of sync with package.json '
+    if (fs.existsSync(lernaPath)) {
+      var lerna = require(lernaPath)
+      if (log.version !== lerna.version) {
+        var errStr = 'CHANGELOG.md out of sync with lerna.json '
+        errStr += '(' + (log.version || log.title) + ' !== ' + lerna.version + ')'
+        return callback(new Error(errStr))
+      }
+    } else if (log.version !== pkg.version) {
+      errStr = 'CHANGELOG.md out of sync with package.json '
       errStr += '(' + (log.version || log.title) + ' !== ' + pkg.version + ')'
       return callback(new Error(errStr))
     }
 
-    var version = pkg.version ? 'v' + pkg.version : null
+    var version = pkg.version ? 'v' + pkg.version : lerna.version ? 'v' + lerna.version : null
```

Comment:
if for some reason `pkg.version` is falsey and `lernaPath` doesn't exist, `lerna.version` will throw an uncaught error. can you construct this in a way that doesn't risk accessing a property on an undefined variable?

Reply:
updated

Reply:
thank you!